packages/core/src/App/Containers/AccountSwitcher/account-switcher-account-list.jsx

```diff
@@ -0,0 +1,107 @@
+
+const AccountList = ({
+    account_type,
+    balance,
+    currency,
+    currency_icon,
+    display_type,
+    has_balance,
+    is_disabled,
+    is_virtual,
+    loginid,
+    onClickAccount,
+    setCurrency,
+    selected_loginid,
+}) => (
+    <>
+        <div
+            id={`dt_${loginid}`}
+            className={classNames('acc-switcher__account', {
+                'acc-switcher__account--selected': loginid === selected_loginid,
+                'acc-switcher__account--disabled': is_disabled,
+            })}
+            onClick={is_disabled ? undefined : onClickAccount}
+        >
+            <span className={'acc-switcher__id'}>
+                <Icon
+                    icon={ currency ? currency_icon : 'IcCurrencyUnknown'}
+                    className={'acc-switcher__id-icon'}
+                    size={24}
+                />
+                <span>
+                    {display_type === 'currency'
+                        ? <CurrencyDisplay is_virtual={is_virtual} currency={currency} />
+                        : <AccountDisplay account_type={account_type} />
+                    }
+                    <div className='acc-switcher__loginid-text'>
+                        {loginid}
+                    </div>
+                </span>
+                {has_balance &&
+                <span className={classNames('acc-switcher__balance', {
+                    'acc-switcher__balance--virtual': is_virtual })}
+                >
+                    {currency &&
+                    <Money
+                        currency={currency}
+                        amount={CurrencyUtils.formatMoney(currency, balance, true)}
+                        should_format={false}
+                    />
+                    }
+                    {!currency &&
+                    <span className='no-currency'>
+                        <Localize i18n_default_text='No currency selected' />
+                    </span>
+                    }
+                </span>
+                }
+            </span>
+        </div>
+        {(!currency && !is_virtual) &&
+            <div
+                className={classNames(
+                    'acc-switcher__account',
+                    'acc-switcher__account-set-currency',
+                )}
+            >
+                <div
+                    className='acc-switcher__account-set-currency-link'
+                    onClick={setCurrency}
+                >
+                    <span>{localize('Select currency')}</span>
+                </div>
+            </div>
+        }
+    </>
+);
+
+const CurrencyDisplay = ({
+    currency,
+    is_virtual,
+}) => {
+    if (is_virtual) {
+        return <Localize i18n_default_text='Demo' />;
+    }
+    if (!currency) {
+        return <Localize i18n_default_text='Real' />;
+    }
+    return currency.toUpperCase();
+};
+
+const AccountDisplay = ({
+    account_type,
+}) => (
+    <React.Fragment>
+        <div>
```

Comment:
do you still need this `<React.Fragment />` ?

Reply:
Nope, probably forgot to remove it after moving a component out. Thanks